runtime/include/chpl-comm-compiler-macros.h

```diff
@@ -158,7 +158,7 @@ chpl_bool chpl_is_node_local(c_nodeid_t node)
 // and then halt the current task.  (The exact behavior is dictated by 
 // chpl_error()).
 static inline
-void chpl_check_local(c_nodeid_t node, int32_t ln, const char* file, const char* error)
+void chpl_check_local(c_nodeid_t node, int32_t ln, const int32_t file, const char* error)
```

Comment:
I see a bunch of places where "const char*" turned into "const int32_t".  I'm not sure what to do about these.  They do effectively say the same thing: that the function won't/can't change the filename.  But they do look a bit odd -- we don't decorate any other int argument decls in this same way, as far as I know. Is the "const" in the new decls really desirable, in your view?